frontend/src/views/Profile.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <v-container fluid>
+    <content-card max-width="600">
+      <template v-slot:title>
+        <v-card-title>
+          <back-button />
+          {{ 'Profil: ' + profile.username }}
+        </v-card-title>
+      </template>
+      <v-col>
+        <api-text-field
+          label="Email"
+          :uri="profile._meta.self"
+          fieldname="mail"
+          :value="profile.mail"
+          :editing="false"
+          required />
+      </v-col>
+    </content-card>
+  </v-container>
+</template>
+
+<script>
+export default {
+  name: 'Home',
+  components: {
+    ApiTextField: () => import('@/components/form/ApiTextField'),
```

Comment:
Würde ich eher nicht dynamisch laden. Das führt zu unnötig vielen Netzwerkrequests. Ich glaubs wir sollten uns für das dynamische Laden auf den Router beschränken. Oder ist die Meinung, dass man das möglichst aufsplittet und dann von Caching profitiert?

Reply:
Ich weiss es nicht. Schauen wir wohl am Besten in #377 an